vagrant-fetch-kubeconfig.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+set -eo pipefail
+
+# The following command sometimes exists with status 1 when using vagrant-libvirt,
+# despite the fact that ssh config was generated successfully.
+vagrant ssh-config > $(pwd)/.ssh_config || true
```

Comment:
You need to run `vagrant ssh-config fedora`. Otherwise it will just fail.

Reply:
Good point. This is connected with multi-distro thing you mentioned in the first comment. Will fix it soon.